### DISCUSSION TITLE
How to disable the 'Send api request'

### DISCUSSION THREAD

**Discussion by rabol** [dedoc/scramble#210] [Q&A]
Hi

Is there a way to remove / disable the 'Send API request' ?

**Comment by rabol** [comment]
One can publish the view and then

```

<elements-api
    apiDescriptionUrl="{{ route('scramble.docs.index') }}"
    router="hash"
    hideTryIt="true"
   />
```

But it removes the whole panel, i would like just to remove the 'button'

**Comment by romalytvynenko** [comment]
Hey @rabol 

Yeah, sadly now you can only remove the entire right panel. Scramble uses Stoplight Elements, so I cannot do anything about it.

I'm working on own UI. Feel free to join waitlist meanwhile: https://dedoc.co/
